config/definitions.yml

```diff
@@ -25,6 +25,12 @@ sle12_common: &sle12_common
     - :server: "http://download.suse.de/ibs/SUSE:/SLE-12-SP2:/Update/standard/"
       :packages: ["hwinfo"]
 
+:sles12-sp3:
+  <<: *sle12_common
+  :yast_url: "http://download.suse.de/ibs/Devel:/YaST:/Head/sle12_sp3/"
```

Comment:
I think we will have to change it in general when we are switching the "master" integration test to SP3. So lets do it at that time.